qa/TL1_paddle_ssd/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="numpy Cython paddle"
+target_dir=./docs/examples/paddle/ssd/
+
+test_body() {
+    # workaround due to errors while pycocotools is in "pip_packages" above
+    pip install -I pycocotools
```

Comment:
Is pycocotools used in this SSD implementation?

Reply:
removed.